tests/release/test_release.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
```

Comment:
These tests were copy and pasted from integration/test_interface.py and the only changes were  reading the noised and unnoised data from fixtures in the release tests.